templates/project/CONTRIBUTING.md

```diff
@@ -93,13 +93,18 @@ php-cs-fixer fix --verbose
 Please note that we try to keep phpdoc to a minimum, so if an `@param` phpdoc
 comment brings nothing more than the type hint and variable name already do,
 it SHOULD be removed. Descriptions are OPTIONAL if you want to document a type.
+If you want to use pseudo-types for static-analysis, prefer the usage of
```

Comment:
pseudo types or generics (with an exception of the @template annotation

Reply:
Why is the `@template` annotation an exception, you could/should also prefix this. AFAIK there was a symfony templating conflict with this annotation.

Reply:
I agree, we should always recommend tu use the phpstan prefix.